### PR TITLE
Fix/ECO-2467-framework-version-updated

### PR DESCRIPTION
**Pull request by Contentstack-AnkitaD**
Angular Framework updated to latest  version:  v15.2.3

### CODE REVIEW

File: .browserslistrc
Comment (surajcontentstack) [inline .browserslistrc:1]: Why is this deleted?

Comment (Contentstack-AnkitaD) [reply .browserslistrc:1]: Its deleted after the package update automatically.

**Review by abhishek305 — APPROVED** [review]

**Review by surajcontentstack — APPROVED** [review]
